apps/api-extractor/src/api/model/ApiMethodSignature.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ApiItemKind } from './ApiItem';
+import { ApiDeclarationMixin, IApiDeclarationMixinOptions } from '../mixins/ApiDeclarationMixin';
+import { ApiFunctionLikeMixin, IApiFunctionLikeMixinOptions } from '../mixins/ApiFunctionLikeMixin';
+import { ApiDocumentedItem, IApiDocumentedItemOptions } from './ApiDocumentedItem';
+import { ApiReleaseTagMixin, IApiReleaseTagMixinOptions } from '../mixins/ApiReleaseTagMixin';
+import { ApiResultTypeMixin, IApiResultTypeMixinOptions } from '../mixins/ApiResultTypeMixin';
+
+/** @public */
+export interface IApiMethodSignatureOptions extends
+  IApiDeclarationMixinOptions,
+  IApiFunctionLikeMixinOptions,
+  IApiReleaseTagMixinOptions,
+  IApiResultTypeMixinOptions,
+  IApiDocumentedItemOptions {
+}
+
+/** @public */
+export class ApiMethodSignature extends ApiDeclarationMixin(ApiFunctionLikeMixin(ApiReleaseTagMixin(
```

Comment:
>ApiMethodSignature [](start = 13, length = 18)

`ApiFunctionSignature`?

Reply:
Here we're using the names the compiler's nomenclature.

---
In reply to: [236468967](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236468967) [](ancestors = 236468967)